plugins/codecs/ndlz/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ set(SOURCES ${SOURCES} ndlz.c ndlz.h ndlz-private.h ndlz4x4.c ndlz4x4.h ndlz8x8.
 
```

Comment:
You don't need `set(SOURCES...)` line anymore, right?

Reply:
And the same goes for other cmake files.

Reply:
Then no plugin source will be distributed. If we want plugins to be in the distribution library, then they must be into SOURCES first.

Reply:
Oh, my mistake.  So if I understand correctly, the updated `$SOURCES` variable is used *outside* of this cmake file, right?  If so, I have no objections.

Reply:
That is what I think. This evening I make sure of it and I confirm you.